tree-sitter.opam

```diff
@@ -7,13 +7,15 @@ dev-repo: "git+https://github.com/returntocorp/ocaml-tree-sitter.git"
 license: "BSD-3-Clause"
 
 build: [
-  ["dune" "build" "-p" name "-j" jobs]
+  ["./configure"]
+  ["make"]
 ]
 
 depends: [
   "alcotest"
   "atdgen"
   "cmdliner"
+  "conf-pkg-config"
```

Comment:
at some point we'll need to write a conf-tree-sitter that does check for tree-sitter and install it if needed, just
like there's a conf-perl, conf-xxx

Reply:
yes!